src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -51,6 +52,7 @@ def exposed_indices(self) -> dict[EntityType, Sorting]:
         return dict(
             activities=Sorting(field_name='activities.activity_id'),
             biosamples=Sorting(field_name='biosamples.biosample_id'),
+            bundles=Sorting(field_name='bundleUuid'),
```

Comment:
Isn't this `bundles.bundleUuid`?

Reply:
I thought so at first as well, but no. 
```
$ http https://service.nadove5.anvil.gi.ucsc.edu/index/datasets?sort=bundleUuid
HTTP/1.1 200 OK
...
{
    "hits": [
        {
...
```

```
$ http https://service.nadove5.anvil.gi.ucsc.edu/index/datasets?sort=bundles.bundleUuid
HTTP/1.1 400 Bad Request
...
{
    "Code": "BadRequestError",
    "Message": "Unknown field `bundles.bundleUuid`"
}
```

The inconsistency is due to the fact that [some fields](https://github.com/DataBiosphere/azul/issues/4573) haven't been pluginified and still use the old HCA names even on AnVIL catalogs.